jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/ShutdownMonitorTest.java

```diff
@@ -134,20 +148,16 @@ public void testStartStopDifferentPortDifferentKey() throws Exception
         assertTrue(!monitor.isAlive());
     }
 
-    /*
-     * Disable these config tests because ShutdownMonitor is a 
-     * static singleton that cannot be unset, and thus would
-     * need each of these methods executed it its own jvm -
-     * current surefire settings only fork for a single test 
-     * class.
-     * 
-     * Undisable to test individually as needed.
+    /**
+     * This test can only be run if the System property
+     * STOP.EXIT has been set. This can either be done in
+     * the IDE, or via the surefire plugin in the pom.
+     * This test expects STOP.EXIT to be FALSE.
      */
-    @Disabled
     @Test
+    @EnabledIf("isStopExitSystemPropertySet")
```

Comment:
Maybe you could simple use `@EnabledIfSystemProperty` ?

Reply:
Done